<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FMEManagerWidget</class>
 <widget class="QWidget" name="FMEManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Server Address</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Parameters</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="versionComboBox"/>
   </item>
   <item row="5" column="0" colspan="5">
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QComboBox" name="workspaceComboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="serverLineEdit"/>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="loadPushButton">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Load / Reload</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resources.qrc">
       <normaloff>:/plugins/DsgTools/icons/refresh.png</normaloff>:/plugins/DsgTools/icons/refresh.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Workspace</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Version</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="sslCheckBox">
     <property name="text">
      <string>Use SSL</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QCheckBox" name="proxyCheckBox">
     <property name="text">
      <string>Use proxy</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
